plugins/serendipity_event_nl2br/serendipity_event_nl2br.php

```diff
@@ -401,547 +401,547 @@ function clean_nl2brtags(&$entry)
+                                'thead', 'tbody', 'tfoot', 'caption', 'ins','del',
+                                'video','audio','title','desc','path','circle',
+                                'ellipse', 'rect', 'line', 'polyline', 'polygon', 'text',
+                                'image', 'g', 'defs'); //includes svg tags
+
+    //paragraphs aren't allowed in these inline elements -> p closes these elements:
+    var $inline_elements = array('b', 'big', 'i', 'small', 'tt', 'abbr',
+                                'acronym', 'cite', 'code', 'dfn', 'em', 'kbd', 'strong',
+                                'samp', 'var', 'a', 'bdo', 'bdi', 'map', 'object',
+                                'q', 'script', 'span', 'sub', 'sup', 'button',
+                                'label', 'select', 'textarea', 's', 'strike' 
+                                );
+
+    var $allowed_p_parents = array('blockquote', 'td', 'div', 'article', 'aside', 'dd',
+                                'details', 'dl', 'dt', 'footer', 'header', 'summary'
+                                );
+
+    const P_END = '</p>';
+    const P = '<p>';
+    const P_NOTOP = '<p class="wl_notop">';
+    const P_NOBOTTOM = '<p class="wl_nobottom">';
+    const P_NOTOPBOTTOM = '<p class="wl_notopbottom">';
+
+    function nl2p($text)
+    {
+        //homogenize tags
+        $text = $this->tag_clean($text);
+        
+        //delete isolation tags from other arrays
+        if ($this->isolationtags)
+        {
+            $this->block_elements = array_diff($this->block_elements,$this->isolationtags);
+            $this->allowed_p_parents = array_diff($this->allowed_p_parents,$this->isolationtags);
+            $this->nested_block_elements = array_diff($this->nested_block_elements,$this->isolationtags);
+            $this->inline_elements = array_diff($this->inline_elements,$this->isolationtags);
+            $this->singleton_block_elements = array_diff($this->singleton_block_elements,$this->isolationtags);
+            $this->ignored_elements = array_diff($this->ignored_elements,$this->isolationtags);
+            $this->isolation_block_elements = array_diff($this->isolation_block_elements,$this->isolationtags);
+        }
+        if (empty($text)) { return ''; }
+        return $this->blocktag_nl2p($text);
+    }
+
+    /** Make sure that all the tags are in lowercase
+     * purge all \n from inside tags
+     * remove spaces in endtags
+     * replace < > with &lt; $gt; for non-tags
+     * tags are split in three parts: 
+     *      tagstart - '<' character
+     *      tagdef - type of tag like 'img'
+     *      style - following content after a space
+     *          isolation by quotes in the style part
+     * @param string text
+     * @return text
+     */
+    function tag_clean($textstring)
+    {
+        $text = str_split($textstring);
+        $tagstart = false;
+        $tagstart_b = false;
+        $tagdef = false;
+        $endtag = false;
```

Comment:
Please try to not mix code formatting and code logic commits; a commit like this is hard to review. Please first commit only the code formatting change and then possible logic changes to better distinguish this, if possible?

Reply:
Sorry for that. I did only the formatting and changed nothing else. I fixed that after I found the formatting rule (indent by spaces) in the code primer. Because I never got invited to work in a team before, I simply don't know about the workflow.

Reply:
All good, I don't wanna come off as picky, it's just that when I try to review code, I try to really see the changes. In an ideal world, we wouldn't mix multiple commits into one as well, but I see the overhead in splitting up this larger commit to multiple variants. 

To go on, please feel free to first make a merge request on just the formatting changes if possible. Then we merge that, and then we can merge the rest of the actual code changes without this overhead of formatting commits?

Reply:
It was just this one and next time, I won't mix, I promise ;-)